Firmware/mmu2_reporting.cpp

```diff
@@ -256,6 +257,7 @@ void ReportErrorHook(CommandInProgress /*cip*/, uint16_t ec, uint8_t /*es*/) {
                 lcd_set_custom_characters();
```

Comment:
Are you going to extract this block into a separate function?
It is the same like `case (uint8_t)ReportErrorHookStates::DISMISS_ERROR_SCREEN:`